test/e2e/daemonset/daemonset.go

```diff
@@ -0,0 +1,34 @@
+package daemonset
+
+import (
+	"context"
+	"time"
+
+	"github.com/onsi/gomega"
```

Comment:
The special `gomega` matchers are imported as :

```
. "github.com/onsi/gomega"
```
So we do `Eventually` instead of `gomega.Eventually`, if the problem is that we cannot declare the new `Eventualy` let's call it `daemonset.GetEventually` in fact it make more sense that `daemonset.Eventually`

Reply:
So, yes, but we can just use it without the '.' here and then use daemonset.Eventually where it is used. Nothing says it *has* to be imported with '.', that's just a convenience. I'm fine with it as it is.